example/declarative.dart

```diff
@@ -1,29 +1,31 @@
 // Copyright 2013 Google. All rights reserved. Use of this source code is
 // governed by a BSD-style license that can be found in the LICENSE file.
 
-library grinder_example_1;
+library grinder.example.declarative;
 
 import 'package:grinder/grinder.dart';
 
-void main([List<String> args]) {
-  task('init', init);
-  task('compile', compile, ['init']);
-  task('deploy', deploy, ['compile']);
-  task('default', null, ['deploy']);
-
+void main(List<String> args) {
```

Comment:
Ok, my plan is to rename `addAnnotatedTasks` to `_discoverTasks` (private for now, we can always expose it later), and add the `grind` method mentioned above.  Then if people need to programmatically add tasks they can just do that before calling `grind`:

```dart
main(args) {
  addTask(someTask);
  grind(args);
}

@Task(...)
...
```